static/js/cohorts/export-manifest.js

```diff
@@ -209,19 +209,23 @@ require([
 
     var update_download_manifest_buttons = function(){
         var is_list = ($('tr:not(:first) input.cohort').length > 0);
+
         var checked_cohorts = $('tr:not(:first) input.cohort:checked').length;
+        var num_inactive = $('tr:not(:first) input.cohort:checked').parent().parent().find('.inactive-data-version').length;
         var num_selected_column =$('.column-checkbox:checked').length;
         var input_cohort_name_len = $('#export-manifest-name').val().length;
 
         if (input_cohort_name_len == 0 || num_selected_column == 0 || (is_list && checked_cohorts == 0)) {
             $('.download-file').attr('disabled', 'disabled');
             $('#get-bq-table').attr('disabled', 'disabled');
         } else {
-            if(is_list && checked_cohorts > 1) {
+            if( (is_list && checked_cohorts > 1) || ((num_inactive>0))) {
                 $('.download-file,.file-manifest').attr('disabled', 'disabled');
-                $('.download-file,.file-manifest').attr('title', 'Only a single cohort\'s file manifest may be downloaded as a file.');
+                $('.download-file,.file-manifest').attr('title', 'Only a single cohort\'s file manifest may be downloaded as a file. A manifest for a cohort with an inactive data version cannot be downloaded as a file.');
                 $('input.bq-manifest').trigger('click');
```

Comment:
This is a bit of a convoluted tooltip; it's better to apply the one that's more relevant, and only that one. If you look below, there's a block where a data attribute is being checked for--inactive-versions is the name I think. That's what needs to be fixed; the template needs to apply it to the row.

Reply:
Did not this block earlier.
On the page I don't see an inactive-versions data attribute on the tr element, but an inactive-data-version class in the versions td element. I think that is why we never reach this block. Will fix that now